core/config/project_settings.h

```diff
@@ -45,9 +45,10 @@ class ProjectSettings : public Object {
 	static const String PROJECT_DATA_DIR_NAME_SUFFIX;
 
 	enum {
-		//properties that are not for built in values begin from this value, so builtin ones are displayed first
+		// Properties that are not for built in values begin from this value, so builtin ones are displayed first
```

Comment:
```suggestion
		// Properties that are not for built in values begin from this value, so builtin ones are displayed first.
```
Should be like this, if you are fixing it.